server/views/pages/offenderSummary.html

```diff
@@ -69,12 +69,27 @@ <h2 class="govuk-heading-m">Offence and sentence</h2>
   </section>
   <section class="govuk-grid-row">
     <h2 class="govuk-heading-m">Sentence Plan</h2>
-    {{ govukTable({
+    {% if sentencePlan %}
+      {{ govukTable({
       firstCellIsHeader: true,
       rows: [
-        [{text: "Date plan started"}, {text: "place holder"}]
+        [{text: "Date plan started"}, {text: sentencePlan.createdDate}]
       ]
     }) }}
+    
+    {% endif %}
+    {{ govukButton({
+      text: "Create Sentence Plan",
```

Comment:
normally text only starts with a capital letter and the other words are all lower case - see https://design-system.service.gov.uk/components/button/